linux/net/rina/dtcp-conf-utils.c

```diff
@@ -1217,3 +1217,4 @@ struct policy * dtcp_rtt_estimator(struct dtcp_config * cfg)
         return cfg->rtt_estimator;
 }
 EXPORT_SYMBOL(dtcp_rtt_estimator);
+
```

Comment:
Sorry, I know this is annoying from my part, but could you remove this blank line?